### ISSUE TITLE
Bulk Upload User Creation

### ISSUE THREAD

**Issue by afolarin** [RADAR-base/ManagementPortal#491]
Would be useful to have a bulk user creation facility for Management Portal. CSV file template or something simple to generate the list then facility to upload and create.

**Comment by yatharthranjan** [comment]
We can use the MP's REST API in an external script (bash or python or similar) to read the CSV and create the users?
And this is for project admins and not for the participants, right?

**Comment by afolarin** [comment]
yes purely admin purposes for speed, only create function required. Also this is non urgent.
